uwblueprint/project-read-frontend
# Add dob questions for students

### Notion ticket link



[Add date of birth questions for students](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=b739b89c29204d25a7a8e7f06af111f3)



### Implementation description

- Added DOB question to the registration form
- Added tests for DOB registration form fields
- Updated date picker component



### Steps to test

1. yarn test
2. yarn start, go to sessions -> add a client -> and see if the date of birth fields work



### What should reviewers focus on?

- Code functionality, robust code design

### Checklist
